jbpm-flow/src/main/java/org/jbpm/process/instance/context/exception/ExceptionScopeInstance.java

```diff
@@ -35,8 +36,10 @@ public ExceptionScope getExceptionScope() {
     public void handleException(String exception, Object params) {
         ExceptionHandler handler = getExceptionScope().getExceptionHandler(exception);
         if (handler == null) {
-            throw new IllegalArgumentException(
-                "Could not find ExceptionHandler for " + exception);
+            throw new IllegalArgumentException("Could not find ExceptionHandler for " + exception);
+        }
+        if( isQueueBased() ) {
+            getProcessInstance().addNewExecutionQueueToStack(false);
```

Comment:
I see this in many places - what does it do? What are the consequences to be set as true or false?